### PR TITLE
Create command that fixes preferred replica imbalance

### PR DESCRIPTION
**Pull request by antban**
Needed this to fix problem of 1 mb in kafka.

### CODE REVIEW

File: bubuku/cli.py [hunk 1]
```diff
@@ -59,6 +62,73 @@ def cli():
     pass
 
 
+@cli.command('preferred-replica-election',
+             help='Do preferred replica election, as command line tool from kafka have a number of limitations. '
+                  'Only partitions, that are improperly allocated will be affected. In case if size of resulting json '
+                  'is too big, it will be split into several parts, and they will be executed one after another.')
+@click.option('--dry-run', is_flag=True, help="Do not apply the changes. Instead just prepare json file(s)")
+@click.option('--max-json-size', type=click.INT, default=512000, help="Maximum size of json data to write to zk",
```
Comment (v-stepanov) [inline bubuku/cli.py]: can you please specify that the size is in bytes

**Comment by v-stepanov** [comment]
everything else looks good

**Comment by v-stepanov** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**